apps/backend/gateway/src/test/resources/application-test.yaml

```
@@ -17,4 +17,4 @@ app:
   permitted:
     routes: /api/v1/user/login,/api/v1/user/register,/api/v1/user/verify,/csrf,/test-csrf,/actuator/*
   access-token-expiration: 3600
-  refresh-token-expiration: 86400
+  refresh-token-expiration: 86400
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)